src/main/scala/com/evolution/scache/CacheMetered.scala

```diff
@@ -96,7 +96,7 @@ object CacheMetered {
         def modify[A](key: K)(f: Option[V] => (A, Directive[F, V])): F[(A, Option[F[Unit]])] =
           for {
             duration <- MeasureDuration[F].start
-            ((a, entryExisted, directive), release) <- cache.modify(key) { entry =>
+            res <- cache.modify(key) { entry =>
```

Comment:
Doesn't this work compile as-is? In Scala 2 it should use `better-monadic-for` and in Scala 3 it works out of the box (at least I was under the impression)

Reply:
it doesn't work by default, to overcome `withFilter` issue one have to put `-source:future` in scalac options. It's really a pity that they didn't included it in LTS version. Anyway, I see that I put such option in `build.sbt` so indeed, I can revert this change